app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] There is issue with your confirmable, devise throws an error of unknow method for nill user, but when I remove confirmable, it works fine, kindly check your devise properl
```
NoMethodError (undefined method `confirmation_token' for #<User id: nil, name: "ceeee", created_at: "2023-12-02 10:57:43.357629000 +0000", updated_at: "2023-12-02 10:57:43.357629000 +0000", email: "c@gmail.com", role: nil>
```